src/frontends/onnx/frontend/src/op/hammingwindow.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "op/hammingwindow.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "utils/common.hpp"
+
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_17 {
+OutputVector hammingwindow(const Node& node) {
+    const auto size = node.get_ng_inputs().at(0);
+    const auto output_datatype = common::get_ngraph_element_type(node.get_attribute_value("output_datatype", 1));
+    const bool periodic = node.get_attribute_value<int64_t>("periodic", 1);
+
+    const ov::PartialShape shape = size.get_partial_shape();
+    const std::vector<size_t> axis_lengths = shape.to_shape();
+
+    element::Type tensor_type;
```

Comment:
You don't need this at all. get_ngraph_element_type returns element::Type or throw an exception if it isn't supported. So, you should use output_datatype directly instead of tensor_type.